deploy/sre-prometheus/100-infra-resizing.PrometheusRule.yaml

```
@@ -115,4 +124,20 @@ spec:
           severity: warning
           namespace: openshift-monitoring
         annotations:
-          message: "The cluster's infrastructure nodes have been consuming excessive memory for 24 hours and should be vertically scaled to support the existing workers. See linked SOP for details."
+      ## If the CPU or Memory related "InfraNodesExcessiveResourceConsumptionSRE" alerts are firing, raise a critical ticket to SRE to scale the infra nodes up
```

```suggestion
      ## If the CPU or Memory related "InfraNodesExcessiveResourceConsumptionSRE" alerts are firing for 2h, raise a critical page to SRE to scale the infra nodes up
```

I'm also ok with deleting this comment completely - I think the rest of the (awesomely clear!) comments paint a clear picture about these alerts